Input.js

```diff
@@ -0,0 +1,111 @@
+export class Input {
+  constructor(inputContainer, displayLineFirst, displayLineSecond) {
+    this.inputContainer = inputContainer;
+    this.firstNumber = null;
+    this.secondNumber = null;
+    this.eqSign = null;
+    this.displayLineFirst = displayLineFirst;
+    this.displayLineSecond = displayLineSecond;
+  }
+  digitButtonPressed = (digit) => {
```

Comment:
The button pressing method should be a part of the Button class not the Input class. This method can call functions on the input class.

Reply:
same goes for `actionButtonPressed`

Reply:
both of the functions use many of variables saved in Input class. Do you want me to save this variables in both places or send them between classes every time theres a button pressed?
i was using this solution since the very beginning of this script and it was never a problem....

Reply:
done